plugins/woocommerce/tests/e2e-pw/tests/shopper/wordpress-post.spec.js

```diff
@@ -6,7 +6,13 @@ const test = baseTest.extend( {
 
 test(
 	'logged-in customer can comment on a post',
-	{ tag: [ '@non-critical' ] },
+	{
+		tag: [
+			'@non-critical',
+			'@skip-on-default-wpcom',
+			'@skip-on-default-pressable',
```

Comment:
The comment design is very different between WPCOM and the rest. It was complicated to put if/else with problematic selectors on WPCOM. Maybe skipping is better option as this test is non critical.